### DISCUSSION TITLE
Making a minimal site with doks

### DISCUSSION THREAD

**Discussion by justsharan** [thuliteio/doks#187] [Q&A] [answered]
I want to create a very basic informational site with doks. I want my "homepage" to be like [this](https://getdoks.org/docs/prologue/introduction/) instead of the default page. How would I do that?

**Comment by epogrebnyak** [comment, marked as answer]
Hi, the appearance of landing page is controlled by index.html, which gives this page a different look. 

I have an example at 
https://gitlab.com/epogrebnyak/doks-howto

Please let know if you'd need a more detailed explanation.

**Comment by johnnymatthews** [comment]
Sorry to dig up an old post. I followed your suggestion @epogrebnyak and replaced the `layouts/index.html` with the [code from your repo](https://gitlab.com/epogrebnyak/doks-howto/-/blob/master/layouts/index.html). However, now the search bar doesn't work on the homepage. It works on any other sub-page, just not the homepage. Any ideas why?

**Reply by h-enk** [reply]
The search script is loaded only when needed — see https://github.com/h-enk/doks/blob/master/layouts/partials/footer/script-footer.html#L77-L79 + https://github.com/h-enk/doks/blob/master/layouts/partials/footer/script-footer.html#L102-L104:

```html
{{ if and (.Site.Params.options.flexSearch) (eq .Section "docs") -}}
  <script src="{{ $index.RelPermalink }}" integrity="{{ $index.Data.Integrity }}" crossorigin="anonymous" defer></script>
{{ end -}}
```

So, try changing it to something like (not tested):

```html
{{ if (.Site.Params.options.flexSearch) -}}
  {{ if or (eq .IsHome) (eq .Section "docs") -}}
    <script src="{{ $index.RelPermalink }}" integrity="{{ $index.Data.Integrity }}" crossorigin="anonymous" defer></script>
  {{ end -}}
{{ end -}}
```

Hope that works

**Reply by johnnymatthews** [reply]
Thanks! That seemed to work, with a little tweeking. For anyone else wondering how to do this, I changed two parts of `layouts/partials/footer/script-footer.html`:

1. Line 77 was:

    ```html
    {{ if and (.Site.Params.options.flexSearch) (eq .Section "docs") -}}
    ```

    I changed it to:

    ```html
    {{ if (.Site.Params.options.flexSearch) -}}
    ```

1. Line 102 was:

    ```html
    {{ if and (.Site.Params.options.flexSearch) (eq .Section "docs") -}}
      <script src="{{ $index.RelPermalink }}" integrity="{{ $index.Data.Integrity }}" crossorigin="anonymous" defer></script>
    {{ end -}}
    ```

    I changed it to

    ```html
    {{ if (.Site.Params.options.flexSearch) -}}
      {{ if or (eq .IsHome) (eq .Section "docs") -}}
          <script src="{{ $index.RelPermalink }}" integrity="{{ $index.Data.Integrity }}" crossorigin="anonymous" defer></script>
      {{ end -}}
    {{ end -}}
    ```

Thinking about it, I could probably just change line 102 to `if and (.Site.Params.options.flexSearch) `.
